tests/integration/telemetry/outboundtrafficpolicy/traffic_registry_only_test.go

```diff
@@ -21,15 +21,16 @@ import (
 
 func TestOutboundTrafficPolicy_RegistryOnly(t *testing.T) {
 	cases := []*TestCase{
-		{
-			Name:     "HTTP Traffic",
-			PortName: "http",
-			Expected: Expected{
-				Metric:          "istio_requests_total",
-				PromQueryFormat: `sum(istio_requests_total{destination_service_name="BlackHoleCluster",response_code="502"})`,
-				ResponseCode:    []string{"502"},
-			},
-		},
+		// Skip this for now due to https://github.com/istio/istio/issues/28139
```

Comment:
would prefer to just skip the metric part but probably NBD since its marked release blocker